src/middlewared/middlewared/plugins/device_/device_info_linux.py

```diff
@@ -22,6 +22,7 @@
 
 class DeviceService(Service, DeviceInfoBase):
 
+    DISK_ROTATION_LOG_CACHE = set()
```

Comment:
`DISK_ROTATION_ERROR_LOG_CACHE` so we see that this is related to the error handling?